src/components/expression/expression.tsx

```diff
@@ -35,10 +36,20 @@ const colorToClassNameMap = {
   danger: 'euiExpression--danger',
 };
 
+const textWrapToClassNameMap = {
+  'break-word': 'euiExpression-breakWord',
```

Comment:
I would leave off creating a class for `break-word` because you aren't actually using that class to add the appropriate styles. Instead, it's the way the default renders. Therefore, you only need to supply an additive class for the `truncate` value.

```suggestion
  'break-word': null,
```